agenta-web/src/components/pages/evaluations/cellRenderers/cellRenderers.tsx

```diff
@@ -186,7 +200,7 @@ export const StatusRenderer = React.memo(
             params.data?.duration || 0,
             runningStatuses.includes(params.value),
         )
-        const {label, color} = statusMapper(token)[params.data?.status.value as EvaluationStatus]
+        const {label, color} = statusMapper(token)(params.data?.status.value as EvaluationStatus)
```

Comment:
out of curiosity, why this change? It does not look like the data structure changed

Reply:
btw, what happens in this case, does the user still can navigate to the result detail view?

Reply:
> out of curiosity, why this change? It does not look like the data structure changed

Updated `statusMapper` function to include a fallback to handle unknown status values returned from BE as "Unknown" rather than break the FE

<img width="1016" alt="Screen Shot 2024-07-04 at 9 32 37 PM" src="https://github.com/Agenta-AI/agenta/assets/99529776/48b31aa9-a4bc-4bde-bb58-b4cb1af422bc">
<img width="1102" alt="Screen Shot 2024-07-04 at 9 33 28 PM" src="https://github.com/Agenta-AI/agenta/assets/99529776/aaf7c09d-6ba7-4a21-9270-0856eb65b03b">



> btw, what happens in this case, does the user still can navigate to the result detail view?

No the user would not be able to navigate to the result detail view. The user can only navigate to the result detail view if the status value is `EVALUATION_FINISHED` or `EVALUATION_FINISHED_WITH_ERRORS`

Reply:
A few comments:

1. Is _red_ the right color for _Unknown_? Why not, but just in case.
2. Is _Result Aggregation Failed_ the right message? The user doesn't know that the individual evaluation scenarios may have succeeded anyhow. <- a UX comment, @mmabrouk
3. In the case of _Result Aggregation Failed_, the user should be able to open the _result detail view_, right ? <- a UI suggestion, @bekossy